sound/soc/sof/ipc4-topology.c

```diff
@@ -1766,7 +1766,11 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 		copier_data->gtw_cfg.config_length += dma_config_tlv_size / 4;
 	}
 
-	dev_dbg(sdev->dev, "copier %s, IPC size is %d", swidget->widget->name, ipc_size);
+	if (deep_buffer_dma_ms > 0)
+		dev_dbg(sdev->dev, "copier %s, IPC size is %d, deep buffer size %u", swidget->widget->name,
+			ipc_size, copier_data->gtw_cfg.dma_buffer_size);
+	else
+		dev_dbg(sdev->dev, "copier %s, IPC size is %d", swidget->widget->name, ipc_size);
```

Comment:
IMHO, we can always print the dma buffer size.

Reply:
Ack, I was inquired several times about how to tell the current deep buff size when experimenting different DEEPBUFFER_FW_DMA_MS.

Reply:
the deepbuffer is only affecting `swidget->id == snd_soc_dapm_aif_in`, in other copiers and w/o DB the buffer size is 2ms worth of bytes.
The question is: what is important? The buffer size in bytes or the length in time? A printing around 1729 if deep_buffer_dma_ms is set might be enough?
Or print only for the host copiers unconditionally?

Reply:
deep_buffer_dma_ms is more clear as an indicator, is the following acceptable?

unconditionally for snd_soc_dapm_aif_in: 
```
[   93.872063] sof-audio-pci-intel-mtl 0000:00:1f.3: copier host-copier.1.playback, deep_buffer_dma_ms 0, dma_buffer_size 384
```
When deepbuf is used:
```
[  156.859504] sof-audio-pci-intel-mtl 0000:00:1f.3: copier host-copier.31.playback, deep_buffer_dma_ms 10, dma_buffer_size 1920
```